include/ck/utility/data_type.hpp

```diff
@@ -992,6 +992,42 @@ inline __host__ __device__ bhalf_t type_convert<bhalf_t, float>(float x)
     return uint16_t(u.int32 >> 16);
 }
 
+// convert fp16 to bf16
+template <>
+inline __host__ __device__ bhalf_t type_convert<bhalf_t, half_t>(half_t x)
+{
+    union
+    {
+        float fp32;
+        uint32_t int32;
+    } u = {static_cast<float>(x)};
+
+    return uint16_t(u.int32 >> 16);
```

Comment:
Is it common practice to simply truncate the lower bits instead of rounding in ML context?

Reply:
MIOpen patch for fix denormal issue  is this way.
Of couse this is simple,there will be some loss of precision, but if the data comes from fp16 and the data is relatively small, the loss may be acceptable